rental.rb

```diff
@@ -0,0 +1,19 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+  end
+
+  def book=(book)
+    @book = book
+    book.rental << self unless book.rental.include?(self)
+  end
+
+  def person=(person)
+    person.rental << self unless person.rental.include?(self)
+  end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, this doesn't work as you could expect:

When you create a new `Rental`, you want that instance to be added to each `Book` and `Person` rentals array. However, this is not happening in your code, the setter methods `book=` and `person=` doesn't get called on initialization.

For this to work, you would have to use those methods inside your initialize method:

```rb
def initialize(date, person, book)
  @date = date
  self.person = person
  self.book = book
end
```

Which doesn't feel that natural. Instead, I would suggest you merge those methods with the initialize one, ending up with something like this:

```suggestion
  def initialize(date, person, book)
    @date = date
    @person = person
    person.rental << self unless person.rental.include?(self)
    @book = book
    book.rental << self unless book.rental.include?(self)
  end
  
  
  
  
  
  
  
```

Please consider implementing this change to apply the both-ways relationship concept correctly! 🤓 🔧

Reply:
Thank you so much for the suggestion